Entities/Items/Bucket/Bucket.as

```diff
@@ -51,7 +51,11 @@ void onTick(CBlob@ this)
 		{
 			this.set_u8("water_delay", water_delay - 1);
 		}
-		else if (point.getOccupied() !is null && point.getOccupied().isMyPlayer() && point.getOccupied().isKeyJustPressed(key_action1) && !this.isInWater())
+		else if (point.getOccupied() !is null && 
+				 point.getOccupied().isMyPlayer() && 
+				 point.getOccupied().isKeyJustPressed(key_action1) && 
+				 !this.isInWater() &&
+				 !point.getOccupied().isKeyPressed(key_inventory)) // prevent splash when doing stuff with inventory
```

Comment:
It might be worth getting the occupied blob once and storing it in a variable instead of doing it four times.

Reply:
Done